io.edgehog.devicemanager.WiFiAP.json

```diff
@@ -0,0 +1,42 @@
+{
+  "interface_name": "io.edgehog.devicemanager.WiFiAP",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "device",
+  "aggregation": "object",
+  "mappings": [
+    {
+      "endpoint": "/ap/channel",
+      "type": "integer",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "The channel over which the client is communicating with the access point."
+    },
+    {
+      "endpoint": "/ap/essid",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Extended Service Set Identification of the current AP"
```

Comment:
what about hidden essid? should we treat "" as undefined/hidden essid?

Reply:
If we decide to collect hidden essid, `""` could be a nice option.